distributed/nanny.py

```diff
@@ -161,6 +164,13 @@ def instantiate(self, comm=None):
 
         Blocks until the process is up and the scheduler is properly informed
         """
+        if self._listen_address:
+            start_arg = self._listen_address
+        else:
+            host = self.listener.bound_address[0]
```

Comment:
Hrm, is there a more consistent way to construct the listening address?  I suppose this problem goes away if we become more consistent about using contact and listening addresses.